### ISSUE TITLE
Bug: Invalid highlight name when trying to add review comment

### ISSUE THREAD

**Issue by trevorwhitney** [pwntester/octo.nvim#226]
## Issue Description

Type: *bug report*

Unable to add review comments.

### Describe what happened (or what feature you want)

I am unable to add review comments.

```
E5108: Error executing lua ...vim/site/pack/packer/start/octo.nvim/lua/octo/colors.lua:165: Invalid highlight name: OctoStatePendingBubble
stack traceback:
[C]: in function 'nvim_get_hl_by_name'
...vim/site/pack/packer/start/octo.nvim/lua/octo/colors.lua:165: in function 'get_background_color_of_highlight_group'
...site/pack/packer/start/octo.nvim/lua/octo/ui/bubbles.lua:22: in function 'make_bubble'
...im/site/pack/packer/start/octo.nvim/lua/octo/writers.lua:487: in function 'write_comment'
...im/site/pack/packer/start/octo.nvim/lua/octo/writers.lua:1312: in function 'write_threads'
...ck/packer/start/octo.nvim/lua/octo/model/octo-buffer.lua:231: in function 'render_threads'
...im/site/pack/packer/start/octo.nvim/lua/octo/reviews.lua:413: in function '_create_thread_buffer'
...im/site/pack/packer/start/octo.nvim/lua/octo/reviews.lua:517: in function 'add_review_comment'
...m/site/pack/packer/start/octo.nvim/lua/octo/mappings.lua:116: in function <...m/site/pack/packer/start/octo.nvim/lua/octo/mappings.lua:115>
...m/site/pack/packer/start/octo.nvim/lua/octo/mappings.lua:9: in function 'on_keypress'
[string ":lua"]:1: in main chunk
```

1. hit enter to dismiss error
1. hit `<space>ca` again to try adding comment again
1. pop-up appears, enter comment, and save with `:w`
1. navigage back to diff, comment window disappears
1. execute `Octo review comments`, see message "No pending comments found"

### Describe what you expected to happen

I would have expected to not see this error and instead be able to add a comment, and have it show up when executing `:Octo review comments`

### How to reproduce it (as minimally and precisely as possible)

1. Octo pr edit 4242
1. Octo review start
1. Navigate to change I want to comment on
1. hit `<space>ca` to add comment
1. see following error:

### Tell us your environment

Neovim nightly and a lot of plugins, I have not tried reproducing on a minimal config yet.
If this seems related to another plugin I can go digging.

**Comment by pwntester** [comment]
Thanks for reporting! Should be fixed by #4de7b07 can you please verify?

**Comment by trevorwhitney** [comment]
Fixed! Thanks so much on all your work on this plugin!
